linalgo/client.py

```diff
@@ -81,6 +87,19 @@ def request(self, url, query_params={}):
             raise Exception(f"Request returned status {res.status_code}")
         return res.json()
 
+    def request_csv(self, url, query_params={}):
+        headers = {'Authorization': f"Token {self.access_token}"}
+        # stream the file
+        with closing(requests.get(url, stream=True, headers=headers, params=query_params)) as res:
+            if res.status_code == 401:
+                raise Exception(f"Authentication failed. Please check your token.")
+            if res.status_code == 404:
+                raise Exception(f"{url} not found.")
+            elif res.status_code != 200:
+                raise Exception(f"Request returned status {res.status_code}")
+            df = pd.read_csv(io.BytesIO(res.content), compression='zip', header=0, sep=',', quotechar='"')
```

Comment:
@nikhiln I think we could just use the csv standard lib here right?

Reply:
@zermelozf Yes, replaced it with standard csv lib.